load_to_rds/lambda_handler.py

```diff
@@ -9,7 +9,20 @@ def lambda_handler(event, session):
     """The lambda handler.
         Event should be a list of inputs from other lambdas."""
 
-    scraped_data = [output["body"]
-                    for output in event]
+    scraped_data = []
+
+    for output in event:
+
+        data = loads(output["body"]["data"])
+
+        if not isinstance(data, dict):
```

Comment:
This can always be changed later, by standardising the returns from each extract lambda.